tools/print-evaluated-rules.js

```diff
@@ -20,6 +27,8 @@ const { ESLint } = require('eslint');
     global.console.log(JSON.stringify(configTypescriptTypechecked, undefined, 4));
     global.console.log('Evaluated Angular rules:');
     global.console.log(JSON.stringify(configAngular, undefined, 4));
+    global.console.log('Evaluated diverged Angular rules:');
```

Comment:
The check isn't for all Angular rules, just the subset that is in Angular extra. Also diverged sounds like it might compare the value of the overwritten rules but it just checks if it is overwritten.
Maybe `'Angular extra rules not overwritten:'`?

Reply:
I would definitely prefer to continue with the term diverged, because it best describes the purpose of the check. Given that, I will include any overrides defined in `angular.ts` and `angular-template.ts`. I didn't consider that initially, because it's a bit of an awkward setup through no fault of our own. I'm not concerned about the values, because if they're equal then they're not divergent and we wouldn't define an override.